gui/default/untrusted/syncthing/core/syncthingController.js

```diff
@@ -1911,36 +1912,48 @@ angular.module('syncthing.core')
             }
         };
 
+        function addFolderInit(recvEnc) {
+            $scope.editingExisting = false;
+            $scope.currentFolder = angular.copy($scope.folderDefaults);
+            if (recvEnc) {
+                $scope.currentFolder.type = "receiveencrypted";
+                $scope.currentFolder.ignorePerms = true;
```

Comment:
Isn't this something the backend should enforce if permission synchronization is not useful for encrypted folders?  Also applies to versioning settings.

I think such things should not be the GUI's responsibility.

Reply:
It is enforced in the backend. The thing is this is a new folder config created in the GUI - it doesn't come from the backend. And the settings must already be correct when presented to the user. See also my answer to your other comment: https://github.com/syncthing/syncthing/pull/7327#discussion_r570083050